### DISCUSSION TITLE
circularity of voronoi cell

### DISCUSSION THREAD

**Discussion by shujing1111** [neperfepx/neper#799] [Q&A] [answered]
Hi,
I am wondering the circularity of the 2D voronoi cells. Like when I compared the voronoi cells with the comand: 1-circularity:lognormal(0.01,0.002) and  1-circularity:lognormal(0.3,0.002).  Why do the voronoi cells exhibit more like circles with 1-circularity:lognormal(0.01,0.002) ? Because I think the larger the mean circularity, the more circle-like? But I got the more circle-like voronoi cells with 1-circularity:lognormal(0.01,0.002).  Thank you

Jing

**Comment by rquey** [comment, marked as answer]
If you use `1-sphericity` (which really is what it means, 1 - *sphericity*, when *sphericity* goes from 0 to 1), then the smaller the mean value, the larger the mean value of the sphericity.
